### PR TITLE
File sources for the sound browser

### PR DESCRIPTION
**Pull request by shadow7483147**
Squashed and separated from patch-1 branch on my fork.

### CODE REVIEW

File: lua/wire/client/sound_browser.lua [hunk 1]
```diff
@@ -30,6 +30,39 @@ local TranslateCHAN = {
 	[CHAN_USER_BASE] = "CHAN_USER_BASE"
 }
 
+local TranslateGameMounts = {
```
Comment (thegrb93) [inline lua/wire/client/sound_browser.lua:33]: You can use the title from engine.GetGames instead can't you?

**Review by thegrb93 — CHANGES_REQUESTED** [review]

File: lua/wire/client/sound_browser.lua (hunk 1 above)
Comment (shadow7483147) [reply lua/wire/client/sound_browser.lua:33]: Yeah, true. I just figured passing the additional string might have been weird if there’s only ever going to be this many games mountable to gmod. I’ll look at it when I get back from work.

**Comment by shadow7483147** [comment]
got one more idea; resolving soundscripts

**Comment by thegrb93** [comment]
I tried it but didn't see the game listed anywhere

**Comment by shadow7483147** [comment]
It's under Path in the details panel.
image
image
image

**Comment by thegrb93** [comment]
They weren't showing up for me. maybe I'll try again in a sec

**Comment by thegrb93** [comment]
I had the wrong branch checked out

**Pull request merged**